api/src/main/java/org/pmiops/workbench/shibboleth/ShibbolethConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.pmiops.workbench.shibboleth;
+
+import java.util.concurrent.TimeUnit;
+import org.pmiops.workbench.auth.UserAuthentication;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.shibboleth.api.ShibbolethApi;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+@Configuration
+public class ShibbolethConfig {
+  public static final String X_APP_ID_HEADER = "X-App-ID";
```

Comment:
Something I learned recently for you to consider: The "X-" prefix is no longer recommended [1].
[1] https://tools.ietf.org/html/rfc6648